### PR TITLE
lab2

### PR DESCRIPTION
**Pull request by ArtyomPeredery**

### CODE REVIEW

File: BlazRobar.html [hunk 1]
```diff
@@ -0,0 +1,63 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+	<meta charset="UTF8">
```
Comment (ArtyomPeredery) [inline BlazRobar.html:4]: Google: Responsive design

File: BlazRobar.html [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+	<title>BlazRobar.com</title>
+	<link href="styles.css" rel="stylesheet">
+</head>
+<body>
+	<header>
+		<a href="">PSD WEBSITES</a>
```
Comment (ArtyomPeredery) [inline BlazRobar.html]: Semantically it's better to use: nav > ul > li > a

File: styles.css [hunk 3]
```diff
@@ -0,0 +1,276 @@
...
+.arrowPic-box{
+	background-color: #5b0fa1;
+	width: 33.333%;
+	justify-content: center;
+	display: inline-flex;
+	align-items: center;
+}
+
+.arrowPic-box img{
+	width: 10% !important;
```
Comment (ArtyomPeredery) [inline styles.css]: Try to avoid this. Google: why

File: BlazRobar.html [hunk 4]
```diff
@@ -0,0 +1,68 @@
...
+		 <ul class="blockpic">
+		<li><a href="">PSD WEBSITES</a></li>
+		<li><a href="">UI KITS</a></li>
+		<li><a href="">PREMIUM PSD FILES</a></li>
+		<Li><a href="">ABOUT</a></Li>
+		<li><a href="">BLOG</a>	</Li>
+		 </ul>
+	 </nav>	
+		 <p class="without-menu">Navigation</p>
+		<div class="hide-menu">&#8801</div>
```
Comment (aliaksandrkrasnou) [inline BlazRobar.html]: Something is broken at this line, please review

Comment (aliaksandrkrasnou) [reply BlazRobar.html]: Probably you forgot ; symbol

**Comment by aliaksandrkrasnou** [comment]
Approved, nice 👍

**Pull request closed without merging**